aws-lc-rs/src/cipher.rs

```diff
@@ -280,7 +329,9 @@ impl Algorithm {
         &self.id
     }
 
-    const fn block_len(&self) -> usize {
+    /// The block length of this cipher algorithm.
+    #[must_use]
```

Comment:
It's usually a good idea to say why the caller should use the result https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-must_use-attribute

```suggestion
    #[must_use = "message goes here"]
```